Base/res/welcome.txt

```diff
@@ -26,11 +26,11 @@ up to your monitor.
 $ /res/icons/16x16/filetype-html.png
 > Connect to the Internet!
 On the Internet, you can correspond through electronic mail (e-mail), get the
-latest news and financial information, and visit Web sites around the world,
+latest news and financial information, and visit websites around the world,
 most of which will make you really angry.
 
 Serenity includes several internet applications, such as an IRC (Internet relay
-chat) client, 4chan browser, telnet server, and basic utilities like ping.
+chat) client, 4chan browser, telnet server, and basic utilities, such as ping.
```

Comment:
Why is this better?

You could come up with a reason why it is, but it's clearly not a typo fix?